server/recceiver/cfstore.py

```diff
@@ -86,17 +121,21 @@ def __commit__(self, TR):
         for pv in delrec:
             if iocid in self.channel_dict[pv]:
                 self.channel_dict[pv].remove(iocid)
-                self.iocs[iocid]["channelcount"] -= 1
+                if iocid in self.iocs:
+                    self.iocs[iocid]["channelcount"] -= 1
```

Comment:
Is the added test here from paranoia, or due to an issue encountered?  I'm a bit nervous about the manual ref. counter, and would like to be clear if there is any issue discovered.